packages/components/src/components/Icon/Icon.stories.tsx

```diff
@@ -21,6 +21,9 @@ const meta: Meta<IconProps> = {
       </View>
     ),
   ],
+  argTypes: {
+    name: { options: Object.keys(IconMap) },
```

Comment:
Turns icon names into a dropdown instead of manually typing them in--helping both draft testing and making it so anyone looking for documentation can actually see all preset icons (instead of it cutting off with "and 40 more...").